easybuild/easyconfigs/b/BLIS/BLIS-1.0-GCC-14.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'BLIS'
+version = '1.0'
```

Comment:
@branfosj Why are sticking to `1.0`, v1.1 was released May 2024: https://github.com/flame/blis/releases/tag/1.1

Reply:
ced7dfd